h2o-r/h2o-package/R/models.R

```diff
@@ -3580,6 +3581,10 @@ h2o.null_dof <- function(object, train=FALSE, valid=FALSE, xval=FALSE) {
 #' @export
 setGeneric("h2o.gainsLift", function(object, ...) {})
 
+#' @rdname h2o.gainsLift
+#' @export
+h2o.gains_lift <- function(object, ...) h2o.gainsLift(object, ...)
```

Comment:
same as Py, I don't understand why we went for dots param in all those plot functions here: there's no validation, and the only reason for their existence seems to be able to pass `xval` when applied to a model object.
In return, nowhere do we seem to validate those dots. Either we allow explicit `xval` param everywhere, emitting a warning if passed non-default value to an object that doesn't support it, or we use dots but at the end validate those, checking that each method received only allowed params (empty dots if metrics, only xval if model).

Reply:
Here I use the `...` just to have the same interface as the generic method that this function should alias.
Rather than fixing this in this PR I think we should instead do it in separate PR (if we decide to) as this pattern (not checking for extra args passed in the dots) is used throughout our R package.

What I did now was to ensure that we pass the dots to all "recursive" calls of `h2o.gainsLift` which makes it eventually fail as the check for the undeclared argument within the dots is checked implicitly by looking at the method signature, e.g., `h2o.gainsLift(model, unused_param)` will call `h2o.gainsLift(model_metrics, unused_param)` and then warns about the unused argument since the signature for `h2o.gainsLift` for model_metrics is just `function(object)`...